Api/SplinterlandsAPI.cs

```diff
@@ -226,8 +226,10 @@ public static async Task<UserCard[]> GetPlayerCardsAsync(string username, string
                     }
                     bool listedOnMarket = (string)card["market_listing_type"] == "RENT" && (string)card["player"] != username ? false : card["market_listing_type"].Type
                         != JTokenType.Null ? true : false;
-                    
-                    return currentUser == username && !cardOnCooldown && !listedOnMarket;
+
+                    bool workingTheLand = card["stake_plot"].Type != JTokenType.Null ? true : false;
```

Comment:
Can be shortened to `bool workingTheLand = card["stake_plot"].Type != JTokenType.Null:` right?

Reply:
I'll merge it and change it myself

Reply:
Yeah, I was matching the previous statement for consistency.